### PR TITLE
Add button component

### LINKED ISSUE #7: [landing page] create button component
https://www.figma.com/file/BfK7HtPsvxZAwZLg1ebz2u/Tracovid?node-id=1193%3A158

### PR DESCRIPTION
**Pull request by ruisaraiva19**
This is related to #7

### CODE REVIEW

File: src/sections/hero/index.jsx [hunk 1]
```diff
@@ -15,6 +28,11 @@ const Hero = () => {
         seu estado durante o surto de COVID-19. Veja a evolução da pandemia na
         sua área de residência.
       </Typography>
+      <Button href={data.site.siteMetadata.appSiteUrl} type="cta">
+        Começar a usar
```
Comment (taniaprocha) [inline src/sections/hero/index.jsx]: You can use `typograhy` as a child of `Button`, and remove font sizes from button css file. Same for other buttons or link.

Comment (ruisaraiva19) [reply src/sections/hero/index.jsx]: Updated!

**Review by taniaprocha — APPROVED** [review]
💯

**Review by ghost — APPROVED** [review]
Good job 👏

**Pull request merged**